applications/GeoMechanicsApplication/tests/test_GeoMechanicsApplication.py

```diff
@@ -106,7 +107,8 @@ def AssembleTestSuites():
                         TestElementaryGroundWaterFlow,
                         KratosGeoMechanicsTransientThermalTests,
                         KratosGeoMechanicsTimeIntegrationTests,
-                        KratosGeoMechanicsTransientPressureLineElementTests
+                        KratosGeoMechanicsTransientPressureLineElementTests,
+                        KratosGeoMechanicsSettlementWorkflow
```

Comment:
Put the new test case in the nightly cases right now, which seem to be run in the GitHub pipeline, should we move this to the 'validation' suite? What is your opinion here @WPK4FEM, @avdg81?

Reply:
Currently the SettlementWorkflow stil is under development, therefore this is a good place for the time being. Once the settlementWorkflow is released and stable, it can be moved as proposed.